Quickstart uninformative links and Grammarly edits (#189)
hubverse-org/hubDocs: docs/source/quickstart-hub-admin/model-metadata-schema.md

# Configuring model metadata  

> For each model that is submitted to the hub, a yaml metadata file ([what is a yaml file?](https://circleci.com/blog/what-is-yaml-a-beginner-s-guide/)) must be added to the `model-metadata` directory by the modeling teams. Click on the hyperlink to read a detailed description of the [`model-metadata` directory](../user-guide/model-metadata.md).  

Reviewer: ```suggestion
For each model that is submitted to the hub, a yaml metadata file ([what is a yaml file?](https://circleci.com/blog/what-is-yaml-a-beginner-s-guide/)) must be added to the `model-metadata` directory by the modeling teams. Details about the `model-metadata` directory can be found in [the model metadata chapter](../user-guide/model-metadata.md).  
```
